api/v1alpha1/selfnoderemediationconfig_webhook_test.go

```diff
@@ -119,6 +119,34 @@ var _ = Describe("SelfNodeRemediationConfig Validation", func() {
 
 	})
 
+	Describe("deleting SelfNodeRemediationConfig CR", func() {
+		var conf *SelfNodeRemediationConfig
+		BeforeEach(func() {
+			conf = createDefaultSelfNodeRemediationConfigCR()
+		})
+
+		When("SelfNodeRemediationConfig CR is default", func() {
+			BeforeEach(func() {
+				conf.Name = "self-node-remediation-config"
+				//Mock deployment namespace
+				_ = os.Setenv("DEPLOYMENT_NAMESPACE", conf.Namespace)
```

Comment:
I'm not sure it's necessary in this case.
The reason is that the value set is the "expected" value, it just so happens that ATM it is required only in this test.
I assume that in the future in case we'll add more tests that requires this variable we will set it at higher level which is common to all the tests instead of only here.